public/data/imran1509.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "Mohd Imran",
+  "type": "Personal",
+  "bio": "I am DevOps Engineer from India. I work with tools like Kubernetes, Docker, Git & GitHub, Ansible, Terraform, Jenkins, AWS, Linux-ubuntu etc. Python & Golang.",
+  "avatar": "https://github.com/imran1509.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/imran1509",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/mohdimran1995/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on twitter",
```

I recommend capitalizing the `t` here. What do you think?
```suggestion
      "name": "Follow me on Twitter",
```